src/cli/lsp.rs

```diff
@@ -329,17 +435,8 @@ mod tests {
         expect_server_initialized(&client.receiver, 1);
 
         let edits: Vec<TextEdit> = expect_response(&client.receiver, 2);
-        assert_eq!(edits.len(), 1);
-        assert_eq!(
-            edits[0],
-            TextEdit {
-                range: Range {
-                    start: Position::new(0, 0),
-                    end: Position::new(0, 14),
-                },
-                new_text: "local x = 1\n".to_string(),
-            }
-        );
+        let formatted = with_edits(contents, edits);
+        assert_eq!(formatted, "local x = 1\n");
```

Comment:
Just as a sanity check, do you mind also expanding the edits list into assertions of what the edits are composed of? To make it clearer what is the output that the byte diff produces